packages/components/src/components/table/table-cell/table-cell-styles.ts

```diff
@@ -1,16 +1,26 @@
 import { getCss } from '../../../utils';
-import { addImportantToEachRule, pxToRemWithUnit, getThemedColors, hostHiddenStyles } from '../../../styles';
+import { addImportantToEachRule, hostHiddenStyles } from '../../../styles';
+import { borderRadiusSmall, spacingFluidSmall } from '@porsche-design-system/utilities-v2';
+import { cssVariableTableRowBackgroundColor } from '../table-row/table-row-styles';
 
 export const getComponentCss = (multiline: boolean): string => {
   return getCss({
     '@global': {
       ':host': {
         ...addImportantToEachRule({
           display: 'table-cell',
-          padding: pxToRemWithUnit(12),
+          padding: spacingFluidSmall,
           margin: 0,
-          borderBottom: `1px solid ${getThemedColors('light').contrastLowColor}`,
           whiteSpace: multiline ? 'normal' : 'nowrap',
+          background: `var(${cssVariableTableRowBackgroundColor}, transparent)`,
+          '&(:first-child)': {
```

Comment:
Hmm, can't we do this on a row level together with the background? Now it seems like there are individual transitions in parallel for every cell within a row when it is hovered.